src/pages/proofs/ProofsTable.tsx

```diff
@@ -53,7 +53,8 @@ export const ProofsTable: React.FC<ProofsTableProps> = ({ filters }) => {
     usePagination(PROOFS_PER_PAGE)
 
   const { data: proofs, isLoading } = useApiQuery(
-    (client) => client.getProofs(filters, offset, limit, orderBy, orderType),
+    (client) =>
+      client.getProofs(filters, offset, limit + 1, orderBy, orderType),
```

Comment:
Why limit + 1?
Some explanation would be nice

Reply:
Currently, it works like this:
1) The current page displays a limit number of proofs.
2) Meanwhile, the page requests limit+1 proofs to determine if there is a next page. If the length of the returned array equals limit + 1, it understands that there definitely is a next page.

Currently, pagination was not displayed because there are 10 proofs on the page, and it receives 10 proofs, so the pagination thinks that there simply is no next page.

Reply:
Currently it works good, and it is how it works in other pages (other table paginations operate with same approach)

Reply:
k, if it solves something. Don't have time to check and propose something better